gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector/policy-studio-debug-inspector.component.ts

```diff
@@ -128,8 +128,13 @@ export class PolicyStudioDebugInspectorComponent implements OnChanges {
 
   ngOnChanges(changes: SimpleChanges) {
     if (changes['inputDebugStep'] || changes['outputDebugStep']) {
-      this.dataSource.data = this.buildTreeNodes();
-      this.treeControl.expandAll();
+      const { nodes, hasErrors } = this.buildTreeNodes();
+      this.dataSource.data = nodes;
+      if (hasErrors && this.treeControl.dataNodes.length > 0) {
+        this.treeControl.expand(this.treeControl.dataNodes[0]);
```

Comment:
maybe now or for future idea if needed 💭 
what about have a list to expand ? or better a expended boolean inside `Node` and resolved  here ? 

this would avoid passing a flag and expend the 1st element arbitrarily (which is the error because we sorted before).